quickstart/deployment.yaml

```diff
@@ -39,5 +39,10 @@ spec:
         env:
           - name: PORT
             value: "8080"
+        # Set resources to be the same as GKE Autopilot defaults http://g.co/gke/autopilot-defaults
```

Comment:
While those are default resources if none specified, they aren't minimum threshold and are probably more than overkill for this workload. Could we do e.g. 250mb of memory and 250m of CPU?